README.md

```diff
@@ -106,15 +106,21 @@ Output:
 
 | Format    | `.tar` | `.zip` | `.gz` | `.xz`, `.lzma` | `.bz`, `.bz2` | `.lz4` | `.sz` | `.zst` |
 |:---------:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
-| Supported | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
+| Supported | ✓ | ✓¹ | ✓² | ✓ | ✓ | ✓ | ✓² | ✓ |
 
-And the aliases: `tgz`, `tbz`, `tbz2`, `tlz4`, `txz`, `tlzma`, `tsz`, `tzst`.
+✓: Supports compression and decompression.
+✓¹: Due to limitations of `.zip`, it doesn't support streaming (de)compression.
+✓²: Supported, and compression runs in parallel.
```

Comment:
```suggestion
✓: Supports compression and decompression.

✓¹: Due to limitations of `.zip`, it doesn't support streaming (de)compression.

✓²: Supported, and compression runs in parallel.
```

Reply:
Oh yeah, Markdown, there's this weird thing where you can force a line break by adding `'\'` at the end of a line, `<br>` works too, because HTML, but if you skip 2 lines, it does add more padding.

Reply:
skipping an extra line looks better than `\` and `<br>` to me in plain text, but if you don't like the extra padding, feel free to switch to one of the alternatives instead